index.html

```diff
@@ -49,5 +49,203 @@ <h4 class="h4-connect">LET'S CONNECT</h4>
         </li>
       </ul>
     </section>
+
+    <section class="cards">
+      <!-- first work section -->
+
+      <div class="card">
+        <img class="img" src="/image/Snapshoot1.png" alt="ID" />
+        <div class="card-contents"></div>
+        <h2>Tonic</h2>
+        <ul class="works">
+          <li>CANOPY</li>
+          <li><img src="/image/counter.png" alt="Counter" /></li>
+          <li>Back End Dev</li>
+          <li><img src="/image/counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="works2">
+          <li><img src="/image/Tag.png" alt="html" /></li>
+          <li><img src="/image/Tag1.png" alt="css" /></li>
+          <li><img src="/image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <button>See Project</button>
+      </div>
+
+      <!-- second work section -->
+
+      <div class="card">
+        <img class="img" src="/image/Snapshoot2.png" alt="ID" />
+        <div class="card-contents"></div>
+        <h2>Mult-Post Stories</h2>
+        <ul class="works">
+          <li>Canopy</li>
+          <li><img src="/image/counter.png" alt="Counter" /></li>
+          <li>Back End Dev</li>
+          <li><img src="/image/counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul >
+          <ul class="works2">
+          <li><img src="/image/Tag.png" alt="html" /></li>
+          <li><img src="/image/Tag1.png" alt="css" /></li>
+          <li><img src="/image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <button>See Project</button>
+      </div>
+
+      <!-- third work section -->
+
+      <div class="card">
+        <img class="img" src="/image/Snapshoot3.png" alt="ID" />
+        <div class="card-contents"></div>
+        <h2>Tonic</h2>
+        <ul class="works">
+          <li>CANOPY</li>
+          <li><img src="/image/counter.png" alt="Counter" /></li>
+          <li>Back End Dev</li>
+          <li><img src="/image/counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="works2">
+          <li><img src="/image/Tag.png" alt="html" /></li>
+          <li><img src="/image/Tag1.png" alt="css" /></li>
+          <li><img src="/image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <button>See Project</button>
+      </div>
```

Comment:
- Kindly add some margin left and right to your card so that it will stick to the Figma  as I show you below :grinning: :grinning::grinning::grinning:

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/192108442-c7fc4ac8-fc62-4c2b-ae26-906faa093c14.png)



## your design 

![image](https://user-images.githubusercontent.com/95297251/192108426-db88908f-102a-4964-9bbf-272a3e377785.png)

Reply:
- Just like the previous reviewer highlighted, your project cards shouldn't take the whole width of the screen, You can look into reducing the width of each card 👍 👍 .